src/main/java/org/candlepin/dto/api/v1/OwnerTranslator.java

```diff
@@ -78,7 +79,9 @@ public OwnerDTO populate(ModelTranslator translator, Owner source, OwnerDTO dest
             .autobindHypervisorDisabled(source.isAutobindHypervisorDisabled())
             .contentAccessMode(source.getContentAccessMode())
             .contentAccessModeList(source.getContentAccessModeList())
-            .lastRefreshed(Util.toDateTime(source.getLastRefreshed()));
+            .lastRefreshed(Util.toDateTime(source.getLastRefreshed()))
+            .anonymous(source.getAnonymous())
```

Comment:
Unrelated to this PR: At some point in the near future outside of this PR when we hit stuff in the Owner again, we should rename the accessors for these new fields to "isAnonymous" and "isClaimed" respectively.